File: 0009-palindrome-number/0009-palindrome-number.java
class Solution 
{
    public boolean isPalindrome(int x)
     {
       int rev=0;
       int temp=x;
       while(temp>0){
        int s=temp%10;
        rev=rev*10+s;
        temp=temp/10;
       }
       if(rev==x){
        return true;
       }
        return false;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        temp = x
        rev = 0
        while temp > 0:
            dig = temp%10
            rev = rev *10 + dig
            temp = temp//10
        if x == rev:
            return True
        else:
            return False
        